README.md

```diff
@@ -61,3 +61,294 @@ julia> logpdf_with_trans(dist, x, false) # ignoring the transformation
 julia> logpdf_with_trans(dist, x, true) # considering the transformation
 -0.7760422307471244
 ```
+
+## `Bijector` interface
+Other than the `logpdf_with_trans` methods, the package also provides a more composable interface through the `Bijector` types. Consider for example the from above with `Beta(2, 2)`.
```

Comment:
"the from above" ==> "above"?

Reply:
Is supposed to be "the one from above"; makes more sense?

Reply:
ah, that makes a lot more sense!